docs/faq.md

```diff
@@ -0,0 +1,25 @@
+---
+id: faq 
+title: FAQ 
+---
+
+<div style='margin-top: -60px'></div>
+<h1 align="center"><b>Welcome To Webaverse</b></h1>
+
+<br/>
+
+These docs will help you create in <a href="https://webaverse.com/" target="_blank" rel="noopener noreferrer">The Webaverse</a>.
+
+The Webaverse ecosystem is built on existing open standards and technologies like WebGL, WebXR, IPFS, and Ethereum.
```

Comment:
Would be good to have a higher level explanation here, perhaps some diagrams/screenshots (though I know that's currently thin on the ground).